src/message_dialog.rs

```diff
@@ -158,16 +154,46 @@ pub enum MessageButtons {
     Ok,
     OkCancel,
     YesNo,
+    YesNoCancel,
     /// One customizable button.
     /// Notice that in Windows, this only works with the feature *common-controls-v6* enabled
     OkCustom(String),
     /// Two customizable buttons.
     /// Notice that in Windows, this only works with the feature *common-controls-v6* enabled
     OkCancelCustom(String, String),
+    /// Three customizable buttons.
+    /// Notice that in Windows, this only works with the feature *common-controls-v6* enabled
+    YesNoCancelCustom(String, String, String),
 }
 
 impl Default for MessageButtons {
     fn default() -> Self {
         Self::Ok
     }
 }
+
+#[derive(Debug, Default)]
```

Comment:
```suggestion
#[derive(Debug, Default, Clone, PartialEq, Eq)]
```
Just to make it easier to use in if statements without pattern matching